assets/dev/js/admin/floating-elements/new-floating-elements.js

```diff
@@ -0,0 +1,52 @@
+var NewFloatingelementsLayout = require( 'elementor-admin/floating-elements/layout' );
```

Comment:
1. Can this be an ES6 module?
2. Can we remove the jQuery?

Reply:
@louiswol94 refactored to use ES6 and remove jQuery